docs/search/get-started-with-search/search-page/log-message-inspector.md

```diff
@@ -0,0 +1,27 @@
+---
+id: log-message-inspector
```

Comment:
I think we are missing some information about the buttons present in the log message inspector panel.
We should add a one-liner for each button present in the panel.
1. Forward log message
2. Previous Log message
3. Share the link
4. Search the field

Reply:
Added info about these buttons.